csharp/examples/IoTDataPipelineExample/Model/SensorData.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace IoTPipelineExample
+{
+    public class SensorData
+    {
+        //Subject-id, Activity Label, Timestamp, x, y, z
+        public int? subjectId { get; set; }
+        public string? activityLabel { get; set; }
+        public DateTimeOffset? timestamp { get; set; }
+        public double? x_Axis { get; set; }
+        public double? y_Axis { get; set; }
+        public double? z_Axis { get; set; }
```

Comment:
Can these names follow standard .NET naming?

```suggestion
        public int? SubjectId { get; set; }
        public string? ActivityLabel { get; set; }
        public DateTimeOffset? Timestamp { get; set; }
        public double? X_Axis { get; set; }
        public double? Y_Axis { get; set; }
        public double? Z_Axis { get; set; }
```